webview-ui/src/components/welcome/WelcomeView.tsx

```diff
@@ -33,10 +33,21 @@ const WelcomeView = () => {
 			<b>To get started, this extension needs an API provider.</b>
 
 			<div style={{ marginTop: "10px" }}>
-				<ApiOptions />
-				<VSCodeButton onClick={handleSubmit} disabled={disableLetsGoButton} style={{ marginTop: "3px" }}>
-					Let's go!
-				</VSCodeButton>
+				<ApiOptions fromWelcomeView />
+				<div style={{ display: "flex", flexDirection: "column", gap: "5px" }}>
+					<VSCodeButton onClick={handleSubmit} style={{ marginTop: "3px" }}>
+						Let's go!
+					</VSCodeButton>
+					{errorMessage && (
+						<span
```

Comment:
We can use tailwind to clean this up a bit:
```tsx
<span className="text-destructive">...</span>
```

I'm not sure if the 12px is the base `--vscode-font-size` value, but you can make the text smaller or larger with `text-sm`, `text-lg`, etc.